packages/mux-player/src/index.ts

```diff
@@ -674,6 +680,55 @@ class MuxPlayerElement extends VideoApiElement {
     this.setAttribute(PlayerAttributes.SECONDARY_COLOR, `${val}`);
   }
 
+  get defaultShowRemainingTime() {
+    return this.hasAttribute(PlayerAttributes.DEFAULT_SHOW_REMAINING_TIME);
+  }
+
+  set defaultShowRemainingTime(val: boolean | undefined) {
+    if (!val) {
+      this.removeAttribute(PlayerAttributes.DEFAULT_SHOW_REMAINING_TIME);
+    } else {
+      this.setAttribute(PlayerAttributes.DEFAULT_SHOW_REMAINING_TIME, '');
+    }
+  }
+
+  get hideDuration() {
+    return this.hasAttribute(PlayerAttributes.HIDE_DURATION);
+  }
+
+  set hideDuration(val: boolean | undefined) {
+    if (!val) {
+      this.removeAttribute(PlayerAttributes.HIDE_DURATION);
+    } else {
+      this.setAttribute(PlayerAttributes.HIDE_DURATION, '');
+    }
+  }
+
+  /**
+   * Get the playback rates applied to the playback rate control.
+   */
+  get playbackRates() {
```

Comment:
should we make playbackRates an AttributeTokenList like controlslist?

Reply:
iiinteresting. I'd lean towards no, at least for now, just because I think that would be clunky for the `set` side of the fence. With this, it's just "set an array". If we wanted to go that route, we'd probably want to at least figure out what that would look like and how it would relate to e.g. https://github.com/muxinc/media-chrome/blob/main/src/js/media-playback-rate-button.js#L33

Reply:
This is particularly relevant for `mux-player-react` fwiw, though we could hypothetically implement the react prop and the custom element prop differently.